build/docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Alpine was chosen by providing a python container less than 100mb
+FROM python:3.10.0b4-alpine3.14
+
+WORKDIR /usr/src/app
+
+# install dependencies
+RUN apk add git npm openjdk11-jre-headless
+
+################
+# TODO: A good practice to apply here in this code is:
+# run container as a non-root user. However, an error is
```

Comment:
Thanks for contributing!

It seems like it should be relatively okay to run commands in this very temporary container as root.  But it still bothers me that there is an error from generateLocalizations.py.

Can you provide details of the error?  I'd like to try to help you solve it so that we can adopt a non-root solution here.

Reply:
I apologize for the delay in providing the error.
Proceeding the contribution, here's the error I get in non-root mode:

```
Successfully tagged docker_shaka-compiler:latest
Creating shaka-compiler ... done
Attaching to shaka-compiler
shaka-compiler    | Traceback (most recent call last):
shaka-compiler    |   File "/usr/src/app/build/all.py", line 150, in <module>
shaka-compiler    |     shakaBuildHelpers.run_main(main)
shaka-compiler    |   File "/usr/src/app/build/shakaBuildHelpers.py", line 374, in run_main
shaka-compiler    |     sys.exit(main(sys.argv[1:]))
shaka-compiler    |   File "/usr/src/app/build/all.py", line 92, in main
shaka-compiler    |     if not localizations.generate(parsed_args.force):
shaka-compiler    |   File "/usr/src/app/build/compiler.py", line 512, in generate
shaka-compiler    |     generateLocalizations.main(['--locales'] + locales)
shaka-compiler    |   File "/usr/src/app/build/generateLocalizations.py", line 255, in main
shaka-compiler    |     with open(args.output, 'wb') as f:
shaka-compiler    | FileNotFoundError: [Errno 2] No such file or directory: '/usr/src/app/dist/locales.js'
shaka-compiler exited with code 1
```
This time, I'll follow the github notifications in order to haven't a delay again.

Reply:
@joeyparrish , Is there any risk of this pull request being closed for update timeout or shouldn't I worry about time? Is there anything I can do in order to help to proceed this PR?